packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -82,6 +82,19 @@ export const configSchema = {
       _description: 'The UUID of the default status for attending a service in the queues eg In Service.',
       _default: 'ca7494ae-437f-4fd0-8aae-b88b9a2ba47d',
     },
+    defaultUrgentPriorityStatus: {
```

Comment:
This should be the default status for queues from the lab. I think we can name it differently